src/sentry/similarity.py

```diff
@@ -444,45 +284,42 @@ def __init__(self, index, aliases, features):
         self.index = index
         self.aliases = aliases
         self.features = features
-        self.__number_format = get_number_format(0xFFFFFFFF)
         assert set(self.aliases) == set(self.features)
 
     def record(self, event):
```

Comment:
Thinking forward to backfill, it might make sense to change this signature to take a sequence of events instead of a single event.